Source/RageCommunity.Library/Extensions/VehicleExtensions.cs

```diff
@@ -95,9 +99,9 @@ public static void RandomiseLicensePlate(this Vehicle vehicle)
             {
                 vehicle.LicensePlate = MathHelper.GetRandomInteger(9).ToString() +
                                        MathHelper.GetRandomInteger(9).ToString() +
-                                       Convert.ToChar(MathHelper.GetRandomInteger(0, 25) + 65) +
-                                       Convert.ToChar(MathHelper.GetRandomInteger(0, 25) + 65) +
-                                       Convert.ToChar(MathHelper.GetRandomInteger(0, 25) + 65) +
+                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 91)) +
+                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 91)) +
+                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 91)) +
```

Comment:
The `GetRandomInteger` values are inclusive, so at 91 you're going to get a [ bracket
```suggestion
                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 90)) +
                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 90)) +
                                       Convert.ToChar(MathHelper.GetRandomInteger(65, 90)) +
```